.circleci/config.yml

```diff
@@ -21,37 +21,47 @@ jobs:
 
     environment:
       # Customize the JVM maximum heap limit
-      JVM_OPTS: -Xmx512m
+      JVM_OPTS: -Xmx1g
 
     steps:
       - checkout
 
       # Download and cache dependencies
       - restore_cache:
          keys:
-           - v1-dependencies-{{ checksum "backend/deps.edn" }}
+           - v1-dependencies-{{ checksum "backend/deps.edn" }}-{{ checksum "frontend/deps.edn"}}
            # fallback to using the latest cache if no exact match is found
            - v1-dependencies-
 
       # run lint
       - run:
           working_directory: "./backend"
-          name: lint
+          name: backend lint
           command: "clj-kondo --lint src/"
 
       # run test
       - run:
           working_directory: "./backend"
-          name: test
+          name: backend test
           command: "clojure -M:dev:tests"
           environment:
             PENPOT_DATABASE_URI: "postgresql://localhost/penpot"
             PENPOT_DATABASE_USERNAME: penpot_test
             PENPOT_DATABASE_PASSWORD: penpot_test
             PENPOT_REDIS_URI: "redis://localhost/1"
 
+      - run:
+          working_directory: "./frontend"
+          name: frontend tests
+          command: |
+            yarn install
+            npx shadow-cljs compile tests
+            node target/tests.js
```

Comment:
This way, the tests execute twice, don't they? Is it necessary the second line?

Reply:
hmm, i think you are right